src/ArgentPonyWarcraftClient.Extensions.DependencyInjection/ArgentPonyWarcraftClient.Extensions.DependencyInjection.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Dan Jagnow, Travis Boatman</Authors>
```

Comment:
@willwolfram18, you should add yourself to the list!

Reply:
Thanks, I'd be happy to do so and appreciate you offering!